src/app/Login.jsx

```diff
@@ -1,59 +1,62 @@
-import React, {PropTypes} from 'react';
+
+/**
+ * Module dependencies
+ */
+
+import React, { PropTypes, Component } from 'react';
 import {connect} from 'react-redux';
 import Radium from 'radium';
 
-import settings from 'settings';
+import { brandColor } from 'settings';
 import color from 'color';
 
 import {login} from 'auth/AuthActions';
 
 import Button from 'components/Button';
 import TextField from 'components/forms/TextField';
 
-const mapStateToProps = state => {
-  return {
-    auth: state.auth,
-    app: state.app
-  };
-};
+/**
+ * Expose login component
+ */
 
-@connect(mapStateToProps)
+@connect(({ auth, app }) => ({ auth, app }))
 @Radium
-class Login extends React.Component {
+export default class Login extends React.Component {
 
   static contextTypes = {
     router: PropTypes.object.isRequired
   }
 
   componentWillMount() {
-    // use react-router to push state?
-    if (!this.props.app.requireLogin || this.props.auth.authorized) {
-      let {router} = this.context;
-      router.push('/');
+    if (this.isAuth()) {
```

Comment:
This usage is a little unclear to me.  It seems that if a user isAuthorized they will be redirected to the root.